src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrDependencyTracker.cs

```diff
@@ -103,6 +105,23 @@ public void UpdateDependencyVersion(VmrDependencyUpdate update)
         gitInfo.SerializeToXml(GetGitInfoFilePath(update.Mapping));
     }
 
+    public bool RemoveRepositoryVersion(string repo)
+    {
+        if (_repoVersions.DeleteVersion(repo))
+        {
+            _repoVersions.SerializeToXml(_allVersionsFilePath);
+        }
+        
+        var gitInfoFilePath = GetGitInfoFilePath(repo);
+        if (_fileSystem.FileExists(gitInfoFilePath))
+        {
+            _fileSystem.DeleteFile(gitInfoFilePath);
+            return true;
+        }
+
+        return false;
```

Comment:
What if there are changes in `_repoVersions.DeleteVersion(repo)`? Shouldn't we return `true`?